Governance: The draupnir longhouse.

Gnuxie opened the pull request:
https://github.com/the-draupnir-project/planning/issues/52

Gnuxie committed:
Governance: The draupnir longhouse.

Gnuxie committed:
Longhouse consultation.

FSG-Cat reviewed (commented):
This probably needs a second round of review and a explanation of intent before formal approval.

Gnuxie committed:
Rename presentations to proposals outside of the longhouse presentation.

Basically, the things individual contributors want to work on are proposals
not presentations. Whereas the main cycle presentation is an actual
presentation of past work and root contributor work.

FSG-Cat reviewed (approved):
Catto is happy enough after talking with Gnuxie
